### PR TITLE
Add "Tweet Now", Toggle for tweet image capabilities and improvements in tweet status.

### LINKED ISSUE #179: Add functionality for add/remove featured image from tweet
### Is your enhancement related to a problem? Please describe.

Sub-task of #59

**Add functionality for add/remove featured image from tweet**
- [x] Include featured image if specified
- [x] If there is a featured image specified show the same thumbnail as the "Featured Image" area
- [ ] Remove featured image from Tweet
- [ ] If featured image was removed but still exists - it can be enabled again
- [ ] Show current featured image with 10% opacity underneath to know what the featured image is 

### Designs

Disabled state with featured image: 
Screen Shot 2019-12-18 at 10 29 32 PM

Enabled - with featured image: 
Screen Shot 2019-12-18 at 10 31 27 PM

Enabled - featured image removed: 
Screen Shot 2019-12-18 at 10 31 55 PM

### Describe alternatives you've considered

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### LINKED ISSUE #180: [Post-Publish Panel] Tweet status improvements
### Is your enhancement related to a problem? Please describe.

Sub-task of #59 

**[Post-Publish Panel] Tweet status improvements**
- [ ] Tweeted status shows date/time and link to the Tweet
- [ ] Post publishing state and icon set appropriately
- [ ] If a post has been Tweeted multiple times, the Tweet entries will stack

### Designs

Published - Tweeted: 
Screen Shot 2019-12-18 at 10 33 48 PM

Published - Disabled: 
Screen Shot 2019-12-18 at 10 34 10 PM

Published - Tweeted multiple times: 
Screen Shot 2019-12-18 at 10 34 58 PM

### Describe alternatives you've considered

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### LINKED ISSUE #181: [AutoTweet Panel]  Add "Tweet now" functionality.
### Is your enhancement related to a problem? Please describe.

Sub-task of #59 

**[AutoTweet Panel]  Add "Tweet now" functionality.**
- [ ] A post can be Tweeted multiple times after publishing - even by a different editor
- [ ] Tweet now will expose controls and allow instant Tweeting
- [ ] Message and settings should persist to allow multi-editor or dedicated social media person workflow

### Designs

Published - Tweeted: 
Screen Shot 2019-12-18 at 10 33 48 PM

Published - Tweet now: 
Screen Shot 2019-12-18 at 10 34 35 PM

### Describe alternatives you've considered

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Sidsector9**
### Description of the Change
This PR following functionalities.

#### 1. Tweet Now Functionality
Tweet now functionality allows users to tweet any time from the post editor (Gutenberg and classic editor). 

| Gutenberg Editor | Classic Editor | 
| --- | --- |
|Screenshot 2022-11-29 at 10 30 46 AM|Screenshot 2022-11-29 at 10 31 12 AM|
|Screenshot 2022-11-29 at 10 30 34 AM|Screenshot 2022-11-29 at 10 31 25 AM|

#### 2. Add functionality for add/remove featured image from tweet
Now user can decide at post level if they want to send featured image in tweet or not.
| Gutenberg Editor | Classic Editor | 
| --- | --- |
|Screenshot 2022-11-29 at 10 36 40 AM|Screenshot 2022-11-29 at 10 38 27 AM|

#### 3. Tweet status improvements
Tweet status now shows multiple logs and the status of each tweet.

| Gutenberg Editor | Classic Editor | 
| --- | --- |
|Screenshot 2022-11-29 at 10 41 24 AM|Screenshot 2022-11-29 at 10 41 42 AM|

Closes #179
Closes #180
Closes #181 

### How to test the Change
1. Configure the plugin.
2. Test each functionality listed above thoroughly.

### Credits

Props @iamdharmesh, @Sidsector9, @jeffpaul, (@linawiezkowiak and @oszkarnagy for the UX/Design work.)

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests pass.

### CODE REVIEW

**Comment by jeffpaul** [comment]
@Sidsector9 what remains to bring this out of draft for review/merge?

**Comment by Sidsector9** [comment]
@jeffpaul I discussed this with @iamdharmesh , some tests are pending before this is ready for merge.

**Comment by iamdharmesh** [comment]
@Sidsector9 I have completed the work here. can you please review it once before we request review from the team.

Thanks.

**Comment by Sidsector9** [comment]
@iamdharmesh thanks for porting the feature to the Classic editor as well. I tested this with:

- Posts that were and were not tweeted through Classic & Gutenberg Editors using `develop`.
- Posts that were and were not tweeted through Classic & Gutenberg Editors using feature branch.
- New posts (tweet) using the feature branch.

Works well 👍

**Comment by jeffpaul** [comment]
@Sidsector9 @iamdharmesh sounds like this is good to merge now, correct?

File: assets/css/admin-autoshare-for-twitter.css [hunk 1]
```diff
@@ -29,8 +30,7 @@
 }
 
 .autoshare-for-twitter-post-status svg {
-	transform: translate3d(0, 3px, 0);
-    margin-right: 5px;
+    margin-right: 4px;
```
Comment (cadic) [inline assets/css/admin-autoshare-for-twitter.css]:
```suggestion
	margin-right: 4px;
```

**Review by cadic — DISMISSED** [review]
LGTM with the small fix in CSS

**Comment by cadic** [comment]
E2E tests are broken, reported in https://github.com/10up/autoshare-for-twitter/issues/211

**Review by iamdharmesh — APPROVED** [review]

**Pull request merged**